### PR TITLE
[feat]: Flask app Setup

### PR DESCRIPTION
**Pull request by Crispy-rw**
- Creating a flask app

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    storage.close()
+
+@app.route('/hello') # example.com/hello/Anthony
+def hello():
+    return 'Hello Crisoy!' # returns hello Anthony!
+
+if __name__ == '__main__':
+    my_host = os.getenv('HBNB_API_HOST')
+    my_port = os.getenv('HBNB_API_PORT')
+    app.run(host=my_host, port=my_port, threaded=True)
```
Comment (EugeneJoe) [inline api/v1/app.py]: Need to convert ENV variable HBNB_API_PORT to integer (just cast it in the app.run call i.e port=int(my_port)

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"""
+
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+import os
+
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
```
Comment (EugeneJoe) [inline api/v1/app.py]: Perhaps we don't need to add the url prefix here as well?

File: api/v1/views/index.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+"""
+
+from flask import jsonify
+from api.v1.views import app_views
+
+
+@app_views.route('/status', methods=['GET'], strict_slashes=False)
+def jsonify_app():
+    """ Function that returns a JSON """
+    return jsonify({"status": "OK"})
```
Comment (EugeneJoe) [inline api/v1/views/index.py:20]: Can we make this print a new line at the end as well?

File: api/v1/app.py (hunk 2 above)
Comment (Crispy-rw) [reply api/v1/app.py]: You are right. I removed it

File: api/v1/app.py (hunk 1 above)
Comment (Crispy-rw) [reply api/v1/app.py]: Done

File: api/v1/views/index.py (hunk 3 above)
Comment (Crispy-rw) [reply api/v1/views/index.py:20]: I have added this config line `app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True` in the app.py which adds a newline at the end and makes the output look nicer.

Comment (EugeneJoe) [reply api/v1/views/index.py:20]: On my side when I curl status it prints the output but without the new line. Perhaps its something on my end

Comment (EugeneJoe) [reply api/v1/views/index.py:20]: vagrant@vagrant-ubuntu-trusty-64:~/AirBnB_clone_v3$ curl -X GET http://0.0.0.0:5000/api/v1/status
{
  "status": "OK"
}vagrant@vagrant-ubuntu-trusty-64:~/AirBnB_clone_v3$


This is how the output looks on my side

**Pull request merged**